src/coreclr/src/debug/createdump/main.cpp

```diff
@@ -5,14 +5,14 @@
 
 #ifdef HOST_WINDOWS
 #define DEFAULT_DUMP_PATH "%TEMP%\\"
-#define DEFAULT_DUMP_TEMPLATE "dump.%d.dmp"
+#define DEFAULT_DUMP_TEMPLATE "dump.%p.dmp"
 #else
 #define DEFAULT_DUMP_PATH "/tmp/"
-#define DEFAULT_DUMP_TEMPLATE "coredump.%d"
+#define DEFAULT_DUMP_TEMPLATE "coredump.%p"
 #endif
 
 const char* g_help = "createdump [options] pid\n"
-"-f, --name - dump path and file name. The pid can be placed in the name with %d. The default is '" DEFAULT_DUMP_PATH DEFAULT_DUMP_TEMPLATE "'\n"
+"-f, --name - dump path and file name. The %p, %e, %h %t format characters are supported. The default is '" DEFAULT_DUMP_PATH DEFAULT_DUMP_TEMPLATE "'\n"
```

Comment:
Is this still the right default, or would it make sense to include some of the new goodness in it now - at least %e (useful in single file apps ?) and maybe %t?

Reply:
I'm worried that someone might depend on the default format.